backend/app/services/rag/pipeline/pandoc.py

```
@@ -0,0 +1,288 @@
+                "-t",
+                "markdown",
+                input_path,
+                "-o",
+                output_path,
+                "--wrap=none",  # Disable line wrapping for better splitting
+            ]
+
+            logger.debug(f"Running Pandoc command: {' '.join(cmd)}")
+
+            # Execute Pandoc
+            result = subprocess.run(
```

Ruff (0.15.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)